seq2science/scripts/get_genome_annotation.py

```diff
@@ -1,41 +1,50 @@
 """
 Script to download genome annotation
 """
-import os
 import contextlib
 
 import genomepy
 
 
 with open(snakemake.log[0], "w") as log:
     with contextlib.redirect_stdout(log), contextlib.redirect_stderr(log):
-        try:
-            provider = snakemake.params.providers[snakemake.wildcards.raw_assembly]["annotation"]
+        # list user plugins
+        active_plugins = genomepy.functions.config.get("plugin", [])
+        # deactivate user plugins
+        genomepy.funcions.manage_plugins("disable", active_plugins)
```

Comment:
This breaks when doing this in parallel, not that I really care tbf :innocent:

Reply:
If we want to do this properly, this should come from Snakemake. And onexit and onerror these should be set back. Perhaps even easier to do the whole thing there?

Reply:
Somewhere here, although that is super confusing this is in loggen :yum: 

https://github.com/vanheeringen-lab/seq2science/blob/develop/seq2science/rules/configuration_logging.smk#L7-L22

Reply:
Anyways, I don't think it really matters

Reply:
all genomepy rules share resource `parallel_downloads=1`, so there should be no parallel genomepy runs.

Reply:
parallel_downloads is set to three though :upside_down_face:

Reply:
(in that three downloads run in parallel0

Reply:
This is to bypass the problem when downloading e.g. two samples, where one is five time the size of the other. This guarantees a more continuous stream of files being downloaded for downstream processing. Anyways, I don't think we really have to worry about these cases

Reply:
its an edge case yea. I've added a genomepy downloads resource too. should be fixed forever now :)